bundlesize.config.json

```diff
@@ -17,8 +17,8 @@
       "maxSize": "750 kB"
     },
     {
-      "path": "packages/shared/dist/umd/index.js",
-      "maxSize": "450 B"
+      "path": "packages/shared/dist/esm/index.js",
+      "maxSize": "75 B"
```

Comment:
I don't see the point of checking the size of a non-bundled file, as this is only the entry, not the final size